src/dataService/pocketSavesService.ts

```diff
@@ -117,10 +72,25 @@ export class PocketSaveDataService {
     'title',
     'user_id',
   ];
-  constructor(context: Pick<IContext, 'apiId' | 'dbClient' | 'userId'>) {
+
+  // For release/toggle flags
+  private flags: Record<string, boolean>;
+  constructor(
+    private context: Pick<
+      IContext,
+      'apiId' | 'dbClient' | 'userId' | 'unleash'
+    >,
+  ) {
     this.apiId = context.apiId;
     this.db = context.dbClient;
     this.userId = context.userId;
+    this.flags = {
```

Comment:
I found msyelf doing that multi-line isEnabled everywhere so I just simplified it here